front/src/views/index.js

```diff
@@ -26,3 +26,23 @@ export {
   default as MyHistoryPreviousTrips
 }
   from '@views/user/history-previous-trips/history-previous-trips.view.vue';
+
+export {
+  default as MyHistoryUpcomingTrips
+}
+  from '@views/user/history-upcoming-trips/history-upcoming-trips.view.vue';
+
+export {
+  default as MyHistoryTripRoute
+}
+  from '@views/user/history-trip-route/history-trip-route.view.vue';
+
+export {
+  default as MyHistoryAddTripRoute
+}
+  from '@views/user/history-trip-add-route/history-trip-add-route.view.vue';
```

Comment:
I beleave that we don't need for creating new route separate view - is is usual route with empty state, isn't it?

Reply:
no history of name - it is just details of routing and it is unnecessary info in name of view

Reply:
no, it's waypoint create/edit form